indico/modules/events/editing/client/js/editing/timeline/StateIndicator.jsx

```diff
@@ -18,33 +21,54 @@ const colors = {
   needs_submitter_changes: 'orange',
 };
 
-export default function StateIndicator({text, circular, tooltip, state}) {
-  const trigger = (
-    <Label size="tiny" color={colors[state]} circular={circular}>
-      {text}
-    </Label>
-  );
+const labelText = {
+  replaced: 'Replaced',
+  needs_submitter_confirmation: 'Needs submitter confirmation',
+  rejected: 'Rejected',
+  accepted: 'Accepted',
+  assigned: 'Assigned',
+  needs_submitter_changes: 'Needs submitter changes',
```

Comment:
Missing i18n; and I'd put this in models.js where we also have `EditableTypeTitles` etc